Set/Supporting Files/AppDelegate.swift

```diff
@@ -25,7 +25,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return UISceneConfiguration(name: "Default Configuration", sessionRole: connectingSceneSession.role)
     }
 
-    func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: Set<UISceneSession>) {
+    private func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: SetGame) {
```

Comment:
looks like you changed an appDelegate method to private?

Reply:
It was an automatic fix for a warning on Xcode.